src/reaper-vault-erc4626.ts

```diff
@@ -36,11 +34,19 @@ export function handleStrategyAdded(event: StrategyAdded): void {
 }
 
 export function handleStrategyReported(event: StrategyReported): void {
-
-  log.info('handleStrategyReported called', []);
+  log.info("handleStrategyReported called", []);
 
   const params = event.params;
   const strategyId = params.strategy.toHexString();
+  const strategy = Strategy.load(strategyId);
+  if (strategy == null) {
```

Comment:
pulling the early return to the front so the remainder of the code in this function doesn't have to be indented